app/models/assessment.rb

```diff
@@ -2,4 +2,9 @@ class Assessment < ApplicationRecord
   belongs_to :student
 
   validates :score, presence: true, length: { maximum: 6 }
+  validates :category, presence: true
+  validates :name, presence: true
+  validates :level, presence: true
+  validates :assessment_type, presence: true
+  validates :date, presence: true
```

Comment:
we could also validate proper pairings between assessment categories, names, and levels. but that's a 'nice to have'.